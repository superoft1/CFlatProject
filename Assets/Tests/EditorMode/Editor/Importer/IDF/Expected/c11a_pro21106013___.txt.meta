fileFormatVersion: 2
guid: a7a083f463b3843428d8bbe8e4c96389
TextScriptImporter:
  externalObjects: {}
  userData: 
  assetBundleName: 
  assetBundleVariant: 
