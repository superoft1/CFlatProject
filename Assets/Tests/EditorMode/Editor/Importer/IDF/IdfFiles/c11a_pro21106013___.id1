  3275     0     1     0  1000     0      0    10   100     30   135    16    16     2
     0     0     0     1     1    25      0     0   201      0    11     0     1    20
    90     1     0   100     0   100 108035     1     0    125     1     0     0     3
     0     1     1     1     1     1      1     1     1      0     0     4     0     0
     0     0     1     1     1     1     55     0     0   1000     0     0     0     0
  8000     0     0     0     0     0      0     0     0      0     0     0     0     0
     0     0     0     0     0     0      0     0     0      0     0     0     0     0
     2    65    30     0     0     0      0     0     0      2 11111     0     1     0
     0     1     0   250     1     0      0   100     0      0     7     0     0     0
     0     0     0     0     0     0      0     0     0      0     0     0     0     0
-102  /pro021106013
  -1  .i*          
-103  /\\LTCPDS\JED
  -1  \3d\iso\sym\s
  -1  ymbol.lib    
-104  /pro021106013
  -1  .bom         
-105  /pdsidf.err  
-108  /pro021106013
  -1  .prt         
-115  /pro021106013
  -1  .sht         
-117  /pro021106013
  -1  .ret         
-122  /\\LTCPDS\JED
  -1  \3d\iso\sym\i
  -1  gs.mld       
-123  /\\LTCPDS\JED
  -1  \3d\iso\detai
  -1  l\jediso.ddf 
-124  /\\LTCPDS\JED
  -1  \3d\iso\detai
  -1  l\jediso.cel 
   3      -6   47900    4800     300       0       0       0       0       0       0       0       0       0
   3     -14   46120    1120     200       0       0       0       0       0       0       0       0       0
   3     -41       0       0       0       0       0       0       0       0       0       0       0       0
   3     -42       0       0       0       0       0       0       0       0       0       0       0       0
   3     -43       0       0       0       0       0       0       0       0       0       0       0       0
   3     -44       0       0       0       0       0       0       0       0       0       0       0       0
   3    -702       0       0       0       0       0       0       0       0       0       0       0       0
   3    -703       0       0       0       0       0       0       0       0       0       0       0       0
-201  E            
-202  N            
-203  W            
-204  S            
-205  EL +         
-206  EL -         
-207   ?           
-213  OPERATOR     
-215  RDF          
-218  MITER        
-222  SLOPE        
-243  FLAT         
-244  TOP          
-245  BOTTOM       
-246  NORTH        
-247  SOUTH        
-248  EAST         
-249  WEST         
-252               
-253               
-255  DWG          
-270               
-271               
-272               
-284               
-289  S.CLASS:     
-290  INSUL.:      
-291  TRACE:       
-293  AREA:        
-294  SCH  :       
-295  PAINT:       
-297               
-305  ME NO.       
-317               
-318               
-359  DIA*LENGTH   
-451               
  -6  8"-PRO-021106
  -1  013-103CS2X-H
  -1  65MM         
  -7  021106013-PRO
  -1  C11A-'       
 -11  103CS2X      
 -14  21/09/11     
 -15  H 65MM       
 -41  C11A         
 -42  UNDEFINED    
 -43  UNDEFINED    
 -44  UNDEFINED    
-942  1            
 -45  UNDEFINED    
-945  021PRO106013 
-946  C11A021PRO106
  -1  013          
-947  C11A         
-948  *C11A021PRO10
  -1  6013*        
 -31  021-G-0013B/$
  -1  SN$8" RFFE 30
  -1  0            
 300         55503       42656         101           0           0           0    0       0,   0,       0,    ,       0       0
 110         75177       10001       52998       75177       10000       53173  128       1,   0,   10000,    ,       0       2
 115             0           0           0           0           0           0   14      30,   0,      12,    ,       0       2
 105         75177       10000       53173       75177       10000       64373  128       3,   0,   10000,FLWN,       0    1011
 -39         3     
  -1  796    1544  
  -1     735    154
  -1  4            
 120         75177       10000       64373       75177       10000       64373  128       0,   0,       0,WW  ,       0       1
 -68  *UNDEFINED*  
 149         75177       10000      173550           0           0           0  128       0,   0,       0,FLOW,       0      10
 100         75177       10000       64373       75177       10000      282727  128       4,   0,   10000,    ,       0    1011
 -39         5     
  -1  416    1544  
  -1     735    154
  -1  4            
 120         75177       10000      282727       75177       10000      282727  128       0,   0,       0,WW  ,       0       1
 -68  *UNDEFINED*  
  55         75177       10000      282727       75177       10000      300506  128       5,   0,   10000,RCBW,       0    1011
   0             0           0           0           0           0           0  160       0,   0,       0,    ,       0       0
 -39         3     
  -1  795    1544  
  -1     735    154
  -1  4            
 120         75177       10000      300506       75177       10000      300506  160       0,   0,       0,WW  ,       0       1
 -68  *UNDEFINED*  
  35         75177       10000      300506       75177       10000      338605  160       6,   0,   10000,ELBW,    9000    1011
 -39         3     
  -1  794    1544  
  -1     734    154
  -1  4            
  36         75177       10000      338605       75177      -28098      338605  160       0,   0,   10000,ELBW,    9000    1011
 120         75177      -28098      338605       75177      -28098      338605  160       0,   0,       0,WW  ,       0       1
 -68  *UNDEFINED*  
 100         75177      -28098      338605       75177     -149997      338605  160       7,   0,   10000,    ,       0    1011
 -39         5     
  -1  414    1544  
  -1     734    154
  -1  4            
 150         75177     -149997      338605           0           0           0  160       8,   0,   10000,    ,       0     502
 -22  TEF2S-1      
 -39         8     
  -1  695    1544  
  -1     734    154
  -1  4            
 150         75177     -149997      338605           0           0           0  160       9,   0,   10000,    ,       0     502
 -22  SPH-C11-061  
 -39         8     
  -1  382    1544  
  -1     734    154
  -1  4            
 150         75177     -149997      338605           0           0           0  160      10,   0,   10000,    ,       0     602
 -22  SH-10-100-300
 -39         8     
  -1  240    1544  
  -1     734    154
  -1  4            
 149         75177     -159449      338605           0           0           0  160       0,   0,       0,FLOW,       0      10
 100         75177     -149997      338605       75177     -168901      338605  160       7,   0,   10000,    ,       0    1011
 -39         5     
  -1  414    1544  
  -1     734    154
  -1  4            
 149         75177     -173911      338605           0           0           0  160       0,   0,       0,FLOW,       0      10
 100         75177     -168901      338605       75177     -178922      338605  160       7,   0,   10000,    ,       0    1011
 -39         5     
  -1  414    1544  
  -1     731    154
  -1  4            
  40         75177     -178922      338605       75177     -178922      338605  160       7,   0,   10000,    ,       0    1001
   0             0           0           0           0           0           0  160       0,   0,       0,    ,       0       0
 -78  SN01_S       
 -39         3    2
  -1  144    1544  
  -1     731    154
  -1  4            
 120         75177     -178922      338605       75177     -178922      338605   24       0,   0,       0,WW  ,       0       1
 -68  *UNDEFINED*  
  41         75177     -178922      338605       90975     -178922      338605   24      11,   0,   10000,SKSW,       0    1001
 120         90975     -178922      338605       90975     -178922      338605   24       0,   0,       0,WW  ,       0       2
 -68  *UNDEFINED*  
 120         90975     -178922      338605       90975     -178922      338605   24       0,   0,       0,BW  ,       0       2
 130         90975     -178922      338605      111475     -178922      338605   24      12,   0,   10000,GGBW,       0    1502
 -78  SN01_S       
 -39         3    1
  -1  890    1544  
  -1    1910    154
  -1  4            
 -37  GTVH         
 150        103173     -178922      338625           0           0           0   24      13,   0,   10000,    ,       0     502
 -22  GST-H1A      
 -39         8     
  -1  735    1544  
  -1    1910    154
  -1  4            
 150        103173     -178922      338625           0           0           0   24      13,   0,   10000,    ,       0     502
 -22  GST-H1A      
 -39         8     
  -1  736    1544  
  -1    1910    154
  -1  4            
 120        111475     -178922      338605      111475     -178922      338605   24       0,   0,       0,WW  ,       0       2
 -68  *UNDEFINED*  
  55        111475     -178922      338605      122903     -178922      338605   24      14,   0,   10000,CPBW,       0    1001
   0             0           0           0           0           0           0   12       0,   0,       0,    ,       0       0
 -39         3    1
  -1  888    1544  
  -1    1910    154
  -1  4            
 -37  SNCW         
 120        122903     -178922      338605      122903     -178922      338605   12       0,   0,       0,WW  ,       0       1
 -68  *UNDEFINED*  
 126        122903     -178922      338605      124172     -178922      338605   12      15,   0,   10000,COSW,       0    1001
 -78  SN01_S       
 -78  SN01_S       
 -39         3    1
  -1  889    1544  
  -1    1909    154
  -1  4            
 120        124172     -178922      338605      124172     -178922      338605   12       0,   0,       0,WW  ,       0       1
 -68  *UNDEFINED*  
 140        124172     -178922      338605      129172     -178922      338605   12      16,   0, 1210000,    ,       0    1002
 -22  021$PT-0542-N
  -1  1            
 -39         6     
  -1  179    1544  
  -1    1909    154
  -1  4            
 -33               
  42         75177     -178922      338605       75177     -178922      338605  160       7,   0,   10000,    ,       0    1001
 149         75177     -189822      338605           0           0           0  160       0,   0,       0,FLOW,       0      10
 100         75177     -178922      338605       75177     -200722      338605  160       7,   0,   10000,    ,       0    1011
 -39         5     
  -1  414    1544  
  -1    2171    154
  -1  4            
 120         75177     -200722      338605       75177     -200722      338605  160       0,   0,       0,WW  ,       0       1
 -68  *UNDEFINED*  
  45         75177     -200722      338605       75177     -221411      338605  160      17,   0,   10000,TEBW,       0    1011
   0             0           0           0           0           0           0  160       0,   0,       0,    ,       0       0
 -22  021-STRT-0013
  -1  B            
 -39         3    1
  -1  796    1544  
  -1    1801    154
  -1  4            
  46         75177     -221411      338605       28890     -221411      338605  160       0,   0,   10000,TEFL,       0    1011
 107         28540     -221411      338605       26135     -221411      338605  160      18,   0, 1210000,FLBL,       0    1002
 -39         3    1
  -1  797    1544  
  -1    1800    154
  -1  4            
 -33               
  47         75177     -221411      338605       75177     -242101      338605  160       0,   0,   10000,TEBW,       0    1011
 120         75177     -242101      338605       75177     -242101      338605  160       0,   0,       0,WW  ,       0       1
 -68  *UNDEFINED*  
 149         75177     -252163      338605           0           0           0  160       0,   0,       0,FLOW,       0      10
 100         75177     -242101      338605       75177     -262226      338605  160       7,   0,   10000,    ,       0    1011
 -39         5     
  -1  908    1544  
  -1    1593    154
  -1  4            
  40         75177     -262226      338605       75177     -262226      338605  160       7,   0,   10000,    ,       0    1001
   0             0           0           0           0           0           0  160       0,   0,       0,    ,       0       0
 -78  SN01_S       
 -39         3    1
  -1  636    1544  
  -1    1590    154
  -1  4            
 120         75177     -262226      338605       75177     -262226      338605   16       0,   0,       0,WW  ,       0       1
 -68  *UNDEFINED*  
  41         75177     -262226      338605       75177     -262226      354578   16      19,   0,   10000,SKSW,       0    1001
 120         75177     -262226      354578       75177     -262226      354578   16       0,   0,       0,WW  ,       0       1
 -68  *UNDEFINED*  
 149         75177     -262226      357828           0           0           0   16       0,   0,       0,FLOW,       0       0
 100         75177     -262226      354578       75177     -262226      361078   16      20,   0,   10000,    ,       0    1001
 -39         5     
  -1  989    1544  
  -1    1890    154
  -1  4            
 120         75177     -262226      361078       75177     -262226      361078   16       0,   0,       0,WW  ,       0       1
 -68  *UNDEFINED*  
 130         75177     -262226      361078       75177     -262226      369228   16      21,   0,   10000,VVSW,       0    1401
 -78  SN01_S       
 -39         3    1
  -1  868    1544  
  -1    1890    154
  -1  4            
 -37  GTVA         
 120         75177     -262226      369228       75177     -262226      369228   16       0,   0,       0,SC  ,       0       1
 150         75177     -262226      362828           0           0           0   16      22,   0,   10000,    ,       0     302
 -22  GST-V1A      
 -39         8     
  -1  715    1544  
  -1    1890    154
  -1  4            
 150         75177     -262226      362828           0           0           0   16      22,   0,   10000,    ,       0     302
 -22  GST-V1A      
 -39         8     
  -1  716    1544  
  -1    1890    154
  -1  4            
  95         75177     -262226      369228       75176     -262226      370606   16      23,   0,   10000,PL  ,       0    1001
 -39         3    1
  -1  869    1544  
  -1    1890    154
  -1  4            
  96         75176     -262226      370606       75177     -262226      371985   16       0,   0,   10000,PL  ,       0    1001
 -33               
  42         75177     -262226      338605       75177     -262226      338605  160       7,   0,   10000,    ,       0    1001
 149         75177     -272288      338605           0           0           0  160       0,   0,       0,FLOW,       0      10
 100         75177     -262226      338605       75177     -282351      338605  160       7,   0,   10000,    ,       0    1011
 -39         5     
  -1  908    1544  
  -1    1590    154
  -1  4            
 120         75177     -282351      338605       75177     -282351      338605  160       0,   0,       0,WW  ,       0       1
 -68  *UNDEFINED*  
 105         75177     -282351      338605       75177     -294150      338605  160      24,   0,   10000,FLWN,       0    1011
 -39         3    1
  -1  634    1544  
  -1    1590    154
  -1  4            
 -37  FLGJ         
 110         75177     -294150      338605       75177     -294500      338605  160      25,   0,   10000,    ,       0       2
  95         75177     -294500      338605       75177     -295800      338605  160      26,   0,   10000,SB2 ,       0    1012
 -39         3    1
  -1  633    1544  
  -1    1590    154
  -1  4            
 -37  TAIL TOP     
  96         75177     -295800      338605       75177     -297100      338605  160       0,   0,   10000,SB2 ,       0    1012
 110         75177     -297100      338605       75177     -297450      338605  160      25,   0,   10000,    ,       0       2
 115             0           0           0           0           0           0   16      27,   0,      16,    ,       0       2
 130         75177     -297450      338605       75177     -343150      338605  160      28,   0,   10000,VVFL,       0    1512
 -39         3    1
  -1  632    1544  
  -1    1590    154
  -1  4            
 -37  ORIENT HANDWH
  -1  EEL WEST     
 110         75177     -343150      338605       75177     -343500      338605  160      25,   0,   10000,    ,       0       2
 115             0           0           0           0           0           0   16      31,   0,      16,    ,       0       2
 105         75177     -343500      338605       75177     -355298      338605  160       1,   0, 1110000,FLWN,       0      11
 -37  FLGB         
 -30  AREA CODE: C1
  -1  1A$LINE: PRO-
  -1  021033002    
 -20  4A842J2C2I   
 -21  GASKET       
  -1      KAMMPROFI
  -1  LE W/FLOATING
  -1  -OR 316L-SS A
  -1  SME CL.300 RF
  -1   RING=316L-SS
  -1   W/GRH-FACED 
  -1  8 " 4.00MM   
 -20  5A300CZU07140
 -21  BOLT AND NUT 
  -1      SBN/FULL-
  -1  THD A193-B7 W
  -1  /A194-2H HVY-
  -1  HEX-NUT UNC-A
  -1  SME S-FN GR-2
  -1  AB, 145.0MM B
  -1  OLT LENGTH   
 -20  3A101FBC2IR  
 -21  FLANGE       
  -1      A105 ASME
  -1   CL.300 WN RF
  -1  -SERR SURF RO
  -1  UGHNESS < 3.2
  -1   MICRO-METER 
  -1  NORMALIZED 8 
  -1  " SCH 60     
 -20  1A113F*C2I   
 -21  PIPE         
  -1      A106-B SM
  -1  LS BE 8 " SCH
  -1   60          
 -20  6E205F*C2KC2I
 -21  CON. REDUCER 
  -1      A234-WPB 
  -1  SMLS BE   10 
  -1  " X 8 " SCH 6
  -1  0 X SCH 60   
 -20  6A205F*C2K   
 -21  90 DEG LR ELB
  -1  OW  A234-WPB 
  -1  SMLS BE   10 
  -1  " SCH 60     
 -20  1A113F*C2K   
 -21  PIPE         
  -1      A106-B SM
  -1  LS BE 10 " SC
  -1  H 60         
 -20  TEF2S-1      
 -21  ERROR READING
  -1   SPECIALTY MA
  -1  TERIAL DESCRI
  -1  PTION LIBRARY
 -20  SPH-C11-061  
 -21  ERROR READING
  -1   SPECIALTY MA
  -1  TERIAL DESCRI
  -1  PTION LIBRARY
 -20  SH-10-100-300
 -21  ERROR READING
  -1   SPECIALTY MA
  -1  TERIAL DESCRI
  -1  PTION LIBRARY
 -20  6K121C*C1HC2G
 -21  S.W. OUTLET  
  -1      A105 ASME
  -1   CL.6000 SW N
  -1  ORMALIZED 1-1
  -1  /2 X 6 " - 36
  -1   "           
 -20  2A100T9C1H   
 -21  GATE VALVE   
  -1      A105  CL.
  -1  800 EXT-BODY 
  -1  SW/PLAIN-M   
  -1  13CR-TR/SEAT-
  -1  STLT BB-OS&Y 
  -1  LOW-EMISSION-
  -1  TP CL.B NORMA
  -1  LIZED 1-1/2 S
  -1  CH160        
 -20  GST-H1A      
 -21  ERROR READING
  -1   SPECIALTY MA
  -1  TERIAL DESCRI
  -1  PTION LIBRARY
 -20  6P123YMC1HC1E
 -21  CON. SWAGED N
  -1  PL  A234-WPB 
  -1  PE WBE SMLS L
  -1  =MSS 1-1/2 X 
  -1  3/4" SCH160 X
  -1   XXS         
 -20  6L100D*C1E   
 -21  FULL COUPLING
  -1      A105 ASME
  -1   CL.9000 SW N
  -1  ORMALIZED 3/4
  -1  "            
 -20  021-PT-0542-N
  -1  1            
 -21  ERROR READING
  -1   SPECIALTY MA
  -1  TERIAL DESCRI
  -1  PTION LIBRARY
 -20  021-STRT-0013
  -1  B            
 -21  STRAINER     
  -1      T-TP A234
  -1  -WPB   BW SML
  -1  S CVR-FLG=ASM
  -1  E CL.300 A105
  -1   RF-SERR SCRN
  -1  =304-SS SURF 
  -1  ROUGHNESS < 3
  -1  .2 MICRO-METE
  -1  R A105=NORMAL
  -1  IZED AS PER D
  -1  ATA SHEET : S
  -1  A-JER-DC021-C
  -1  YSE-223501 10
  -1   " SCH 60    
 -81  103CS2X      
 -20  3B101B*C2KR  
 -21  BLIND FLANGE 
  -1      A105 ASME
  -1   CL.300 RF-SE
  -1  RR SURF ROUGH
  -1  NESS < 3.2 MI
  -1  CRO-METER NOR
  -1  MALIZED 10 " 
 -20  6K121C*C1FC2G
 -21  S.W. OUTLET  
  -1      A105 ASME
  -1   CL.6000 SW N
  -1  ORMALIZED 1 "
  -1   X 6 " - 36 "
 -20  1A112L*C1F   
 -21  PIPE         
  -1      A106-B SM
  -1  LS PE   1 " S
  -1  CH160        
 -20  2A108A6C1F   
 -21  GATE VALVE   
  -1      A105  CL.
  -1  800 SW/NPT-F 
  -1    13CR-TR/SEA
  -1  T-STLT BB-OS&
  -1  Y LOW-EMISSIO
  -1  N-TP CL.B NOR
  -1  MALIZED 1 "  
 -20  GST-V1A      
 -21  ERROR READING
  -1   SPECIALTY MA
  -1  TERIAL DESCRI
  -1  PTION LIBRARY
 -20  6Q102A*C1F   
 -21  PLUG         
  -1      A105 NPT 
  -1  RND-HD NORMAL
  -1  IZED 1 "     
 -20  3A101FRC2KR  
 -21  FLANGE       
  -1      A105 ASME
  -1   CL.300 WN RF
  -1  -SERR SURF RO
  -1  UGHNESS < 3.2
  -1   MICRO-METER 
  -1  W/JACK-SCR NO
  -1  RMALIZED 10 "
  -1   SCH 60      
 -20  4A842J2C2K   
 -21  GASKET       
  -1      KAMMPROFI
  -1  LE W/FLOATING
  -1  -OR 316L-SS A
  -1  SME CL.300 RF
  -1   RING=316L-SS
  -1   W/GRH-FACED 
  -1  10 " 4.00MM  
 -20  3P235JBC2KR  
 -21  SPECTACLE BLI
  -1  ND  A516-70 A
  -1  SME CL.300 SE
  -1  RR-FCG SURF R
  -1  OUGHNESS < 3.
  -1  2 MICRO-METER
  -1   10 "        
 -20  5A300CZU10190
 -21  BOLT AND NUT 
  -1      SBN/FULL-
  -1  THD A193-B7 W
  -1  /A194-2H HVY-
  -1  HEX-NUT UNC-A
  -1  SME S-FN GR-2
  -1  AB, 190.0MM B
  -1  OLT LENGTH   
 -20  2A207YQC2K   
 -21  GATE VALVE   
  -1      A216-WCB 
  -1  ASME CL.300 R
  -1  F-SERR   13CR
  -1  -TR/SEAT-STLT
  -1   BB-OS&Y LOW-
  -1  EMISSION-TP C
  -1  L.B 10 "     
 -20  5A300CZU10160
 -21  BOLT AND NUT 
  -1      SBN/FULL-
  -1  THD A193-B7 W
  -1  /A194-2H HVY-
  -1  HEX-NUT UNC-A
  -1  SME S-FN GR-2
  -1  AB, 165.0MM B
  -1  OLT LENGTH   
 -20  5A300CZU07140
 -21  BOLT AND NUT 
  -1      SBN/FULL-
  -1  THD A193-B7 W
  -1  /A194-2H HVY-
  -1  HEX-NUT UNC-A
  -1  SME S-FN GR-2
  -1  AB, 140.0MM B
  -1  OLT LENGTH   
 -20  5A300CZU10160
 -21  BOLT AND NUT 
  -1      SBN/FULL-
  -1  THD A193-B7 W
  -1  /A194-2H HVY-
  -1  HEX-NUT UNC-A
  -1  SME S-FN GR-2
  -1  AB, 160.0MM B
  -1  OLT LENGTH   
 999
