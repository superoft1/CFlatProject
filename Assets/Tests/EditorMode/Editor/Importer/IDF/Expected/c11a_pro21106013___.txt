WeldNeckFlange, 3A101FBC2IR, (0.0, 0.0, -1.0), (-55503.8, 42656.1, 101.6), False
(-55503.8, 42656.1, 101.5)
(-55503.8, 42656.1, 101.6)
FreeVertex: 1
Pipe, 1A113F*C2I, (0.0, 0.0, 1.0), (-55503.8, 42656.1, 102.7), False
(-55503.8, 42656.1, 101.6)
(-55503.8, 42656.1, 103.8)
FreeVertex: 0
ConcentricPipingReducer, 6E205F*C2KC2I, (0.0, 0.0, 1.0), (-55503.8, 42656.1, 103.9), False
(-55503.8, 42656.1, 104.0)
(-55503.8, 42656.1, 103.8)
FreeVertex: 0
Elbow90, 6A205F*C2K, (0.0, 1.0, 0.0), (-55503.8, 42656.1, 104.4), False
(-55503.8, 42656.1, 104.0)
(-55503.8, 42655.7, 104.4)
FreeVertex: 0
Pipe, 1A113F*C2K, (0.0, -1.0, 0.0), (-55503.8, 42655.1, 104.4), False
(-55503.8, 42655.7, 104.4)
(-55503.8, 42654.5, 104.4)
FreeVertex: 0
Pipe, 1A113F*C2K, (0.0, -1.0, 0.0), (-55503.8, 42654.4, 104.4), False
(-55503.8, 42654.5, 104.4)
(-55503.8, 42654.3, 104.4)
FreeVertex: 0
Pipe, 1A113F*C2K, (0.0, -1.0, 0.0), (-55503.8, 42654.3, 104.4), False
(-55503.8, 42654.3, 104.4)
(-55503.8, 42654.2, 104.4)
FreeVertex: 0
WeldOlet, 1A113F*C2K, (-1.0, 0.0, 0.0), (-55503.8, 42654.2, 104.4), False
(-55503.8, 42654.2, 104.4)
(-55503.8, 42654.2, 104.4)
(-55503.9, 42654.2, 104.4)
FreeVertex: 0
GateValve, 2A100T9C1H, (-1.0, 0.0, 0.0), (-55504.0, 42654.2, 104.4), False
(-55504.1, 42654.2, 104.4)
(-55503.9, 42654.2, 104.4)
FreeVertex: 1
EccentricPipingReducer, 6P123YMC1HC1E, (1.0, 0.0, 0.0), (-55504.2, 42654.2, 104.4), False
(-55504.1, 42654.2, 104.4)
(-55504.2, 42654.2, 104.4)
FreeVertex: 2
Coupling, 6L100D*C1E, (-1.0, 0.0, 0.0), (-55504.2, 42654.2, 104.4), False
(-55504.2, 42654.2, 104.4)
(-55504.2, 42654.2, 104.4)
FreeVertex: 2
Pipe, 1A113F*C2K, (0.0, -1.0, 0.0), (-55503.8, 42654.1, 104.4), False
(-55503.8, 42654.2, 104.4)
(-55503.8, 42654.0, 104.4)
FreeVertex: 0
Tee, 021-STRT-0013B, (0.0, -1.0, 0.0), (-55503.8, 42653.8, 104.4), True
(-55503.8, 42653.6, 104.4)
(-55503.8, 42654.0, 104.4)
(-55503.5, 42653.8, 104.4)
FreeVertex: 1
BlindFlange, 3B101B*C2KR, (-1.0, 0.0, 0.0), (-55503.3, 42653.8, 104.4), False
(-55503.3, 42653.8, 104.4)
FreeVertex: 1
Pipe, 1A113F*C2K, (0.0, -1.0, 0.0), (-55503.8, 42653.5, 104.4), False
(-55503.8, 42653.6, 104.4)
(-55503.8, 42653.4, 104.4)
FreeVertex: 0
WeldOlet, 1A113F*C2K, (0.0, 0.0, 1.0), (-55503.8, 42653.4, 104.4), False
(-55503.8, 42653.4, 104.4)
(-55503.8, 42653.4, 104.4)
(-55503.8, 42653.4, 104.5)
FreeVertex: 0
Pipe, 1A112L*C1F, (0.0, 0.0, 1.0), (-55503.8, 42653.4, 104.6), False
(-55503.8, 42653.4, 104.5)
(-55503.8, 42653.4, 104.6)
FreeVertex: 0
GateValve, 2A108A6C1F, (0.0, 0.0, 1.0), (-55503.8, 42653.4, 104.7), False
(-55503.8, 42653.4, 104.7)
(-55503.8, 42653.4, 104.6)
FreeVertex: 0
OpenSpectacleBlank, 6Q102A*C1F, (0.0, 0.0, 1.0), (-55503.8, 42653.4, 104.7), False
(-55503.8, 42653.4, 104.7)
(-55503.8, 42653.4, 104.7)
FreeVertex: 1
Pipe, 1A113F*C2K, (0.0, -1.0, 0.0), (-55503.8, 42653.3, 104.4), False
(-55503.8, 42653.4, 104.4)
(-55503.8, 42653.2, 104.4)
FreeVertex: 0
WeldNeckFlange, 3A101FRC2KR, (0.0, -1.0, 0.0), (-55503.8, 42653.1, 104.4), False
(-55503.8, 42653.1, 104.4)
(-55503.8, 42653.2, 104.4)
FreeVertex: 0
OpenSpectacleBlank, 3P235JBC2KR, (0.0, -1.0, 0.0), (-55503.8, 42653.0, 104.4), False
(-55503.8, 42653.0, 104.4)
(-55503.8, 42653.1, 104.4)
FreeVertex: 0
GateValve, 2A207YQC2K, (0.0, -1.0, 0.0), (-55503.8, 42652.8, 104.4), False
(-55503.8, 42652.6, 104.4)
(-55503.8, 42653.0, 104.4)
FreeVertex: 1
FreeVertexTotal: 10
